.github/workflows/nix.yaml

```
@@ -46,11 +46,11 @@ jobs:
   build:
     strategy:
       matrix:
-        system: [x86_64-linux, aarch64-darwin]
+        system: [aarch64-linux]
         isMain:
           - ${{ contains(github.ref, 'main') }}
         exclude:
-          - system: aarch64-darwin
+          - system: x86_64-linux
```

actionlint (1.7.4)

value "x86_64-linux" in "exclude" does not match in matrix "system" combinations. possible values are "aarch64-linux"

(matrix)